src/renderer/state/route/logics.ts

```diff
@@ -289,38 +296,37 @@ const viewUploadsLogic = createLogic({
         requests: UploadRequest[];
         fileIds: string[];
       };
-      const fileIdsAsFiles: UploadRequest[] = await Promise.all(
-        fileIds.map(async (fileId: string) => {
-          const [labkeyFileMetadata, customMetadata] = await Promise.all([
-            labkeyClient.selectFirst<LabKeyFileMetadata>(
-              LK_SCHEMA.FMS,
-              "File",
-              RELEVANT_FILE_COLUMNS,
-              [LabkeyClient.createFilter("FileId", fileId)]
-            ),
-            mmsClient.getFileMetadata(fileId),
-          ]);
-          return {
-            ...labkeyFileMetadata,
-            fileId,
-            customMetadata,
-            file: {
-              originalPath: labkeyFileMetadata.localFilePath as string,
-              fileType: labkeyFileMetadata.fileType,
-            },
-          };
-        })
-      );
-      const uploadsToView = convertUploadRequestsToUploadStateBranch(
-        [...requests, ...fileIdsAsFiles],
-        state
+      const metadataForFiles = await Promise.all(fileIds.map(async (fileId) => await Promise.all([
```

Comment:
I think this is equivalent to just
```
      const metadataForFiles = await Promise.all(fileIds.map(fileId => Promise.all([
      ...
```
Because `return await foo();` is the same as `return foo();` for an async function `foo` (async functions return promises, and `return await x;` means "return a promise that resolves to whatever the promise `x` resolves to)